### ISSUE TITLE
Time not shown correctly, without entering the place name

### ISSUE THREAD

**Issue by Manveer695** [fossasia/susi_skill_data#241]
If we ask SUSI 'what time is it', false time is shown.

**Would you like to work on the issue?**
Yes, working on it.

**Comment by Manveer695** [comment]
SUSI replies with location as berlin, even if I am using it from India. 
<img width="518" alt="myplace" src="https://user-images.githubusercontent.com/20307535/28488564-980a1454-6eca-11e7-8ec6-545bea289bc3.PNG">

@Orbiter Can we get the location of the user from the server, so that we can tell him the time at his place. It can help us in many other skills too.

Till now I have tried query 'where am I':
<img width="313" alt="whereami" src="https://user-images.githubusercontent.com/20307535/28488555-5ebc58e2-6eca-11e7-9910-1d0659b48c85.PNG">

which shows the location of the user in its answer. But, I don't think we can fetch the location from this answer and store it in variables when making a skill in SUSI dream. If we are able to store the latitude and longitude in variables, we can make a call to google maps api to get the name of the place at that latitude and longitude, which can be further used in an api call to wolfram alpha with query as "time in *", where * is the place name fetched from google maps API. Hence, getting the answer.

**Comment by Orbiter** [comment]
The only way to identify the location of the user from the server is an IP mapping, which is not always very accurate. Do you now a service which gives lon/lat from a given IP?

**Comment by Manveer695** [comment]
@Orbiter Yes sir, [this](http://freegeoip.net) will help in that. When you will visit it, it will by default tell you your location according to your IP. Also, for example you may visit http://freegeoip.net/json/github.com will tell you the ip of the device related to github.com or http://freegeoip.net/json/14.141.33.67 will give you the place of this IP address. Thanks for helping me out.

**Comment by LoveChauhan-18** [comment]
A PR has been submitted to fix this issue.

The solution ensures that time queries without a location no longer return incorrect results. Instead, the skill now handles such cases appropriately and maintains consistent behavior.

Please review the PR. I am open to suggestions and improvements.
